apps/pie-docs/src/engineers/guidelines/devices.json

```diff
@@ -44,7 +44,7 @@
                   "text": "Samsung Galaxy Tab"
               },
               {
-                  "text": "S7 to latest"
+                  "text": "S8 to latest"
```

Comment:
Do we need to update the iPad support at all? Given the logic we use for the iOS support (The oldest device available that supports the last 3 major versions of iOS), we probably should do the same for iPad OS?

Given this page, the minimum supported device would be iPad Air (3rd Generation) to latest - https://support.apple.com/en-gb/guide/ipad/ipad213a25b2/18.0/ipados/18.0

I only raise this because I know for a fact the iPad market share (and I suspect our traffic) dwarfs any competing Android Tablet, so found it interesting that the docs currently state only latest iPad is supported.

I know tablet in general accounts for a very smaller percentage of web traffic, but at mentioned, I suspect the majority will be iPad users.